hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/HandlerUtility.java

```diff
@@ -169,6 +169,7 @@ static HederaFunctionality functionalityForType(final DataOneOfType transactionT
             case NODE_CREATE -> HederaFunctionality.NODE_CREATE;
             case NODE_UPDATE -> HederaFunctionality.NODE_UPDATE;
             case NODE_DELETE -> HederaFunctionality.NODE_DELETE;
+            case TOKEN_CANCEL_AIRDROP -> HederaFunctionality.TOKEN_CANCEL_AIRDROP;
```

Comment:
I am confused if this is including `TokenCancelAirdrop` changes or not 🙂 . We should add it everywhere or remove from this PR

Reply:
Summarizing it just to have it as a history here: we commented on this with Neeha on the daily. This PR does not include the cancel logic. The changes related to the cancel are needed because we are using the same protos for both PRs and in order to be able to build this branch we need to add the TOKEN_CANCEL_AIRDROP to some of the classes.